src/modules/GameScreen/GameScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { block } from 'bem-cn';
+
+import GameReactComponent from 'components/GameReactComponent/GameReactComponent';
+import { GameState } from 'game/types';
+import { Button } from 'components/ui';
+import introImage from 'assets/img/game/intro.png';
+import winImage from 'assets/img/game/win.png';
+import lossImage from 'assets/img/game/loss.png';
+import './game-screen.scss';
```

Comment:
❗️
Разные по смыслу блоки стоит отделять пустой строкой
```suggestion
import lossImage from 'assets/img/game/loss.png';

import './game-screen.scss';
```